impeller/aiks/aiks_unittests.cc

```diff
@@ -1939,5 +1910,22 @@ TEST_P(AiksTest, DrawPaintAbsorbsClears) {
   ASSERT_EQ(picture.pass->GetClearColor(), Color::CornflowerBlue());
 }
 
+Picture BlendModeSaveLayerTest(BlendMode blend_mode) {
```

Comment:
Add this to an anonymous namespace to avoid collisions.

Reply:
Done.